Fitness/Controllers/FilterViewController.swift

```diff
@@ -112,13 +117,12 @@ class FilterViewController: UIViewController {
         classTypeInstructorDivider.backgroundColor = .fitnessLightGrey
         contentView.addSubview(classTypeInstructorDivider)
 
-        classTypeDropdownData = DropdownData(dropStatus: .up, titles: [], ids: [], completed: false)
+        classTypeDropdownData = DropdownData(dropStatus: .up, titles: [], completed: false)
 
-        AppDelegate.networkManager.getGymClassDescriptions { (gymClassDescriptions) in
+        AppDelegate.networkManager.getClassNames { (classNames) in
 
-            for gymClassDescription in gymClassDescriptions {
-                self.classTypeDropdownData.titles.append(gymClassDescription.name)
-                self.classTypeDropdownData.ids.append(gymClassDescription.id)
+            for className in classNames {
+                self.classTypeDropdownData.titles.append(className)
```

Comment:
I think you can just do `self.classTypeDropdownData.titles.append(contentsOf:...)`